### PR TITLE
Feat/endTrip initial commit

### PR DESCRIPTION
**Pull request by rckchen**
-added html to display users involved in currentTrip
-hard coded data in currentTrip.js for testing purposes with db.json
-created endTrip files
-createTrip Spec made

### CODE REVIEW

File: client/app/currentTrip/currentTrip.html [hunk 1]
```diff
@@ -1,6 +1,10 @@
 <div>
+  <div ng-model="data">
```
Comment (tmizoroki) [inline client/app/currentTrip/currentTrip.html]: do we need an ng-model here in order for this to work?

File: client/app/currentTrip/currentTrip.js [hunk 2]
```diff
@@ -7,7 +7,8 @@
 
     $scope.hasTrip = function () {
       Trip.hasTrip( function (data) {
-        $scope.data = data;
+        $scope.data = data[0];
```
Comment (tmizoroki) [inline client/app/currentTrip/currentTrip.js]: We need to figure out a way for hasTrip to send the correct trip back. I did the same type of thing to test expenses page with the getExpenses function where I just grabbed the index at zero.

File: test/client/createTripSpec.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+    createController();
+  }));
+
+  afterEach(function () {
+    $httpBackend.verifyNoOutstandingExpectation();
+    $httpBackend.verifyNoOutstandingRequest();
+  });
+
+  it('should have a link property on the $scope', function () {
```
Comment (tmizoroki) [inline test/client/createTripSpec.js]: forgot to change this it heading

File: test/client/createTripSpec.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    expect($scope.trip).to.be.an('object');
+  });
+
+  it('should have a addLink method on the $scope', function () {
```
Comment (tmizoroki) [inline test/client/createTripSpec.js]: and this too

File: test/client/createTripSpec.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
+describe('ShortenController', function () {
```
Comment (tmizoroki) [inline test/client/createTripSpec.js]: Forgot to change describe heading

File: client/app/currentTrip/currentTrip.html (hunk 1 above)
Comment (rckchen) [inline client/app/currentTrip/currentTrip.html]: im not sure let me try it out.

Comment (rckchen) [inline client/app/currentTrip/currentTrip.html]: your understanding is correct. thanks for that

**Pull request merged**